setup.py

```diff
@@ -72,6 +72,8 @@
     "google-generativeai>=0.4.0",
     "zhipuai",
     "litellm",
+    "nbclient",
```

Comment:
Do the example-level dependencies need to be added to `minimal_requires`?

Reply:
to support notebook service function(not just for codeact agent), it need to be added here